### ISSUE TITLE
Tracking on-street infrastructure design consultants

### ISSUE THREAD

**Issue by samsmith** [cyclestreets/cyclescape#290]
Just chatting to Martin, I had the suggestion that cyclescape should include fields listing the various named contractors that we know of in projects. And we can then give them their own nice little pages, listing their projects, and what has had to be done to fix the ones that they designed a couple of years ago etc.

Given that they're often doing multiple junctions elsewhere, the results of that may form a nice little webpage for future accountability (and some companies who do well can point at it, highlighting those that do badly).

cc @mvl22

**Comment by mvl22** [comment]
Example: campaigning about low-quality design proposed for Milton Road, Cambridge (2017) has exposed that the developers have used a traffic model that fails to include cyclists. Attaching a warning to other schemes / planning applications involving this developer would be useful for alerting campaigners to look out for similar problems within future issues.

**Comment by samsmith** [comment]
If company numbers are stored in some structured form, that would allow linking via the OpenCorporates data/APIs to others in corporate groups (ie, you find subsidiaries):
	https://www.opencorporates.com


Another example is the developer in this thread - who will come back:
	https://camcycle.cyclescape.org/threads/3253#message_43388





best,
Sam



> On 23 Jul 02017, at 21:16, Martin <notifications@github.com> wrote:
> 
> Example: campaigning about low-quality design proposed for Milton Road, Cambridge (2017) has exposed that the developers have used a traffic model that fails to include cyclists. Attaching a warning to other schemes / planning applications involving this developer would be useful for alerting campaigners to look out for similar problems within future issues.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 

--  
@smithsam

**Comment by samsmith** [comment]
Another example would be a post-mortem 3 years on for the Catholic Church junction, and which parts have been shown to fail, and the specific companies that gave assurances that they would work. e.g. 
	https://twitter.com/RTaylorUK/status/891317351890780160



If the same design is used elsewhere, the question is what lessons have been learnt? and why are they new…

**Comment by nikolai-b** [comment]
@mvl22, is this something that I can start work on?  I tend to agree that tagging is probably an easy start.  I've fixed https://github.com/cyclestreets/cyclescape/issues/103 so that should help make tagging consultants easier.

**Comment by mvl22** [comment]
Yes. We probably need to have a chat to discuss what this will look like in UI terms in practice.
